ocs_ci/deployment/deployment.py

```diff
@@ -802,14 +802,22 @@ def verify_pvs_created(num_workers):
     assert pv_json['items'], (
         f"No PVs created but we are expecting {num_workers}"
     )
+
+    # check number of PVs created
+    num_pvs = len(pv_json['items'])
+    assert num_pvs == num_workers, (
```

Comment:
OSD's didn't come into picture in this function. We are only checking and make sure  number of PVs created should be equal to the number of workers.

Reply:
Maybe I am missing something (please tell me if I do), but in line 769 in this file (https://github.com/red-hat-storage/ocs-ci/blob/8eb4a1b26c1d686fa66ddd25cec1e28f54616329/ocs_ci/deployment/deployment.py#L769), we update the localvolume CR with the device paths. For i3 case, each worker has 2 paths:
file:///home/Elad/Pictures/Screenshot%20from%202020-05-26%2013-55-12.png

Reply:
this check has been there in https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/deployment/deployment.py#L783

coming to this function doesn't know about i3 or LSO, based on parameter num_workers , it will check

Reply:
As per our discussion, changed the parameter which causes the confusion.